mirascope/cohere/embedders.py

```diff
@@ -58,7 +58,7 @@ def embed(self, inputs: list[str]) -> CohereEmbeddingResponse:
         )
         start_time = datetime.datetime.now().timestamp() * 1000
         embed = co.embed
-        if len(self.configuration.llm_ops) > 0:
+        if self.configuration.llm_ops:  # pragma: no cover
```

Comment:
do we need this if we check in the `get_wrapped_call` function?

Reply:
oh i missed this one good catch